tsl/src/planner.c

```diff
@@ -49,6 +51,21 @@ is_dist_hypertable_involved(PlannerInfo *root)
 	return false;
 }
 
+#if PG14_GE
+static int osm_present = -1;
```

Comment:
```suggestion
static bool osm_present = false;
```
Nit. Shouldn't this be `bool` ??

Reply:
this is tri-state, since we want to cache the result of  the check for this backend, -1 = not yet checked